include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -181,9 +190,10 @@ SECTIONS
 
     GROUP_START(RAMABLE_REGION)
 
-#if defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
+#if defined (CONFIG_SOC_SERIES_NRF51X) ||	\
+	defined(CONFIG_SOC_SERIES_STM32F0X) && !defined(CONFIG_IS_BOOTLOADER)
```

Comment:
Parenthesis preferred to make grouping clear (and avoid mistakes).

    #if (defined(CONFIG_SOC_SERIES_NRF51X) || \
          defined(CONFIG_SOC_SERIES_STM32F0X)) \
         && !defined(CONFIG_IS_BOOTLOADER)